terraform/Jenkinsfiles/TF_Scaffold.groovy

```diff
@@ -1,9 +1,10 @@
 String region = "eu-west-2"
- 
+
 Map <String, Map<String, String>> componentImageBranch = [
-  OneOneOne: [ecrRepo: "111",   branch: "master"],
-  nhais:     [ecrRepo: "nhais", branch: "develop"],
-  nhais_responder: [ecrRepo: "nhais-fake-responder", branch: "origin-develop"]
+  OneOneOne:       [ecrRepo: "111",                  branch: "master"],
+  nhais:           [ecrRepo: "nhais",                branch: "develop"],
+  nhais_responder: [ecrRepo: "nhais-fake-responder", branch: "origin-develop"],
+  gp2gp:           [ecrRepo: "gp2gp",                branch: "develop"]
```

Comment:
the table here is used to query ECR for latest image from 'master' branch. For gp2gp the 'master' branch is called 'main'. Without this terraform will try to set ` 067756640211.dkr.ecr.eu-west-2.amazonaws.com/gp2gp:null` as the image which will obviously fail

Reply:
Sorted out.